src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -75,6 +75,33 @@ public void testGetUser_byId_notFound() {
     Assert.assertNull(resultUser);
   }
 
+  @Test
+  public void testGetUsers_byUsername_found() {
+    List<User> resultUsers = userStore.getUsers("username");
+    Assert.assertEquals(3, resultUsers.size());
+    assertEquals(USER_ONE, resultUsers.get(0));
+    assertEquals(USER_TWO, resultUsers.get(1));
+    assertEquals(USER_THREE, resultUsers.get(2));
+  }
+
```

Comment:
How about one more test here to make sure partial strings are matched? For your sample users in this file, "USER_T" for example should return two results: "USER_TWO" and "USER_THREE".